frontend/src/hooks/review/useGetReviewList/index.ts

```diff
@@ -1,32 +1,31 @@
-import { useSuspenseQuery } from '@tanstack/react-query';
+import { useSuspenseInfiniteQuery } from '@tanstack/react-query';
 
 import { getReviewListApi } from '@/apis/review';
 import { REVIEW_QUERY_KEY } from '@/constants';
 
 const useGetReviewList = (groupAccessCode: string, reviewRequestCode: string) => {
-  const { data, isLoading, error, isSuccess } = useSuspenseQuery({
+  const { data, fetchNextPage, hasNextPage, isLoading, isSuccess } = useSuspenseInfiniteQuery({
     queryKey: [REVIEW_QUERY_KEY.reviews],
-    queryFn: () => getReviewListApi(groupAccessCode, reviewRequestCode),
+    queryFn: ({ pageParam }) =>
+      getReviewListApi({
+        groupAccessCode,
+        reviewRequestCode,
+        lastReviewId: pageParam === 0 ? null : pageParam, // 첫 api 요청 시, null 값 보내기
+      }),
+
+    initialPageParam: 0,
+    getNextPageParam: (data) => {
+      return data.lastReviewId;
+    },
     staleTime: 1 * 60 * 1000,
   });
 
-  // NOTE: 무한스크롤 관련 코드 일단 주석 처리
-  // const { data, fetchNextPage, hasNextPage, isLoading, error } = useInfiniteQuery({
-  //   queryKey: [REVIEW_QUERY_KEY.reviews],
-  //   queryFn: ({ pageParam }) => getReviewListApi(pageParam),
-  //   initialPageParam: GROUP_ACCESS_CODE,
-  //   getNextPageParam: (data) => {
-  //     return data.lastReviewId ? data.lastReviewId : undefined; // 마지막 리뷰 ID가 없을 경우 undefined 반환
-  //   },
-  // });
-
   return {
```

Comment:
```suggestion
cosnt result = useSuspenseInfiniteQuery({...});

return result
```
이렇게 하면 useSuspenseInfiniteQuery의 결과물?을 원하는대로 꺼내서 쓸 수 있어요.

Reply:
와우! 이런 방법은 모르고 있었는데 고마워요!